### DISCUSSION TITLE
FAQ: Talos 1.9 and Linux 6.12

### DISCUSSION THREAD

**Discussion by smira** [siderolabs/talos#9743] [General]
Linux 6.12 release was cut yesterday, and it is going to be the next LTS release.

At this moment, neither of third-party Linux kernel modules (drbd, zfs and NVIDIA) build with Linux 6.12 tree. 

Some of them have fixes coming, or patches available, but zfs at the moment only provides compatibility up to Linux 6.11.

So unless things change in a week or two, Talos 1.9 is going to ship with Linux 6.6, and Talos 1.10 will pick up Linux 6.12.

**Comment by runningman84** [comment]
why not ship with 6.11?

**Reply by smira** [reply]
Talos always ships LTS only versions, as we need them to updated for the lifecycle of Talos 1.9 (for example) which is around 4-5 months. Linux 6.11 will be discontinued well before that.

**Comment by smira** [comment]
**UPDATE**: we're most probably going to land Linux 6.12 in Talos 1.9, with ZFS either disabled or in experimental status.

**Comment by smira** [comment]
Talos 1.9 is going to ship with Linux 6.12, but probably without `zfs` until `zfs` has a release compatible with Linux 6.12
